optimum/exporters/onnx/__main__.py

```diff
@@ -115,41 +122,65 @@ def main():
             f"At least  {onnx_config.DEFAULT_ONNX_OPSET} is required."
         )
 
-    onnx_inputs, onnx_outputs = export(
-        model,
-        onnx_config,
-        args.opset,
-        args.output,
-    )
+    if args.atol is None:
+        args.atol = onnx_config.ATOL_FOR_VALIDATION
+        if isinstance(args.atol, dict):
+            args.atol = args.atol[task.replace("-with-past", "")]
+
+    if model.config.is_encoder_decoder and args.for_ort:
+        encoder_model = model.get_encoder()
+        encoder_onnx_config = onnx_config.get_encoder_onnx_config(encoder_model.config)
+
+        decoder_model = model.get_decoder()
```

Comment:
Does it handle heads after the decoder?

Reply:
This line is just used to get the decoder config from the model. And the entire model is used to export the `decoder+heads` in lines below.

The export uses `encoder_outputs` as input which would ensure that all modules after the `encoder` is present in the exported `decoder` model.

Reply:
Nice!